frontend/src/components/common/wrappers/AuthorizationWrapper.js

```diff
@@ -43,6 +43,7 @@ const propTypes = {
   isMajorMine: PropTypes.bool,
   inDevelopment: PropTypes.bool,
   inTesting: PropTypes.bool,
+  tab: PropTypes.bool,
   children: PropTypes.element.isRequired,
```

Comment:
Are you doing anything with this tab prop? or forgot to remove?

Reply:
Yeah, good catch. I have removed it.